sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StateStoreSuite.scala

```diff
@@ -279,7 +280,7 @@ class StateStoreSuite extends StateStoreSuiteBase[HDFSBackedStateStoreProvider]
     val timeoutDuration = 1.minute
 
     quietly {
-      withSpark(new SparkContext(conf)) { sc =>
+      withSpark(SparkContext.getOrCreate(conf)) { sc =>
```

Comment:
Why do we need this change ?

Reply:
Because now we use a shared spark session, creating a new spark context will have conflict.

Reply:
I don't mean to override the config here, the reason why I need this is because I extends SharedSparkSession, which already have an active spark context, if I don't modify the test, it will throw an error when trying to create a new spark context.